src/utils.js

```diff
@@ -0,0 +1,93 @@
+// user selects one of these options when first adding an item to a list
+export const radioButtonOptions = [
+  {
+    option: 'soon',
+    value: 7,
+    defaultChecked: true,
+  },
+  {
+    option: 'kind of soon',
+    value: 14,
+    defaultChecked: false,
+  },
+  {
+    option: 'not soon',
+    value: 30,
+    defaultChecked: false,
+  },
+];
+
+//used to normalize item names when added by a user
+export const removePunctuation = (string) => {
+  const punctuationlessString = string
+    .toLowerCase()
+    .replace(/[^\w\s]|_/g, '')
+    .split(' ')
+    .join('');
+
+  return punctuationlessString;
+};
+
+// general variables used for time calculations
+export const currentTime = Date.now();
```

Comment:
It seems that this function will run when this constant is set, and not later. Is that intended? (variable name says currentTime, but it's really timeWhenThisVariableWasSet)
Did you mean to do something like:
function getCurrentTime() {
  return Date.now();
}
(or arrow function variant, which would be cleaner)
I haven't tested this, so I could be completely off :)